nodejs/build/common.nix

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
Did this come from Nixpkgs or you? If from nixpkgs, we should probably say at the top.